python/console/console_output.py

```diff
@@ -37,6 +37,7 @@
 
 class writeOut(QObject):
     ERROR_COLOR = "#e31a1c"
+    ERROR_STYLE_INDEX = 16
```

Comment:
genuinely naive question: why 16 ? Is that arbitrary or matches something expected elsewhere... ? Might be worth a code comment

Reply:
Because `QsciLexerPython` uses style codes up to 15. It does not define a LAST_USED_STYLE or USER_STYLE value though, so it is kind of arbitrary (although QsciScintilla has been stagnant for about 9years so it seems unlikely to change at this point). I'll add the comment.